src/Cardano/Shell/Presets.hs

```diff
@@ -0,0 +1,316 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Ah, a bit bittersweet but if it's local to this module only, fine.

Reply:
I can probably take it out, just putting it on hold until I fix the other issues.

Reply:
Removed.